### PR TITLE
Quests - 21030446

### PR DESCRIPTION
**Pull request by MirhasVida**

### CODE REVIEW

File: Translations/ko-KR/quest.json [hunk 1]
```diff
@@ -626,30 +626,30 @@
...
+		"desc": "구식 소형 경순부대의 개장 재편성 : 改 텐류형 경순 텐류 改2 및 타츠타 改2 2척으로 구성된 정예 제18전대를 재편성하라!"
 	},
 	"195": {
 		"code": "A90",
 		"name_jp": "精強「第十七駆逐隊」を編成せよ！",
-		"name": "Form the Elite 17th Destroyer Division!",
+		"name": "정예의 강력한 제17구축대를 편성하라!",
 		"desc_jp": "改装甲型駆逐隊編成任務：対空兵装を軸に大幅強化改装した「磯風乙改」「浜風乙改」、そして<br>対潜兵装を改装強化した「浦風丁改」「谷風丁改」による、精強「第十七駆逐隊」4隻を編成せよ！",
-		"desc": "Have Isokaze B Kai, Hamakaze B Kai, Urakaze D Kai and Tanikaze D Kai in your fleet."
+		"desc": ""개장 갑형 구축대 편성 임무 : 대공 병장를 중심으로 대폭 강화 개장한 이소카제 乙改, 하마카제 乙改, <br>그리고 대잠 병장을 개장 강화한 우라카제 丁改, 타니카제 丁改로 정예의 강력한 제17구축대 4척을 편성하라!"
```
Comment (myangelkamikaze) [inline Translations/ko-KR/quest.json]: There's an extra " at the start.

Comment (MirhasVida) [reply Translations/ko-KR/quest.json]: Solved!

**Comment by MirhasVida** [comment]
Solved!

**Comment by myangelkamikaze** [comment]
Are you still making more changes or should I merge?

**Comment by MirhasVida** [comment]
I'm gonna sync my fork so merge it please

**Pull request merged**